### PR TITLE
feat: add nginx.1.21.4 patch

### PR DESCRIPTION
**Pull request by fesily**

### CODE REVIEW

**Review by spacewander — COMMENTED** [review]
Could you submit a PR to apisix-build-tools
and update https://github.com/api7/apisix-nginx-module/blob/becad3f90293f24698c863c496b0501eb84a1433/.github/workflows/ci.yml#L32 to point to your branch so we can verify the patches in the CI?

**Comment by fesily** [comment]
> Could you submit a PR to apisix-build-tools and update
> 
> https://github.com/api7/apisix-nginx-module/blob/becad3f90293f24698c863c496b0501eb84a1433/.github/workflows/ci.yml#L32
> 
> to point to your branch so we can verify the patches in the CI?

Ok, I fix it

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
 
       - name: Install
         run: |
-          wget https://raw.githubusercontent.com/api7/apisix-build-tools/master/build-apisix-base.sh
+          wget https://raw.githubusercontent.com/fesily/apisix-build-tools/master/build-apisix-base.sh
```
Comment (spacewander) [inline .github/workflows/ci.yml:32]: Let's submit a PR to apisix-build-tools too. Thanks!

**Review by spacewander — APPROVED** [review]
Please update ci.yml back before we can merge it

**Review by tzssangglass — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**